spec.md

```diff
@@ -164,49 +202,100 @@ echo "SPDX-License-Identifier: MIT" > file.txt
 # REUSE-IgnoreEnd
 ```
 
-### In-line snippet comments
-
-If a copyright and/or licensing info is to apply only to a certain snippet
-instead of the whole file, SPDX snippet tags SHOULD be used (as defined in [SPDX
-Specification, Annex H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
-
-Such an annotated snippet block MUST start with `SPDX-SnippetBegin` to mark its
-beginning and end with `SPDX-SnippetEnd` to mark the snippet's end.
-
-Do note that SPDX snippet tags MUST start with `SPDX-Snippet`, meaning that the
-correct copyright notice in a snippet is `SPDX-SnippetCopyrightText`.
-
-Example:
-
-```
-# SPDX-SnippetBegin
-# SPDX-License-Identifier: MIT
-# SPDX-SnippetCopyrightText: 2022 Jane Doe <jane@example.com>
-
-{$snippet_code_goes_here}
-
-# SPDX-SnippetEnd
+### REUSE.toml
+
+Copyright and Licensing Information MAY be associated with a file through a
+`REUSE.toml` file. The intended use case of this method is large directories where
+including a comment header in each file (or in `.license` companion files) is
+impossible or undesirable.
+
+A `REUSE.toml` file MAY be located in any directory, and can cover files that are
+within its directory or deeper. You MAY have multiple `REUSE.toml` files in
+different directories.
+
+The `version` key (REQUIRED) MUST have an integer value representing the schema
+version of the file. This specification describes version 1 of `REUSE.toml`.
+
+Each `[[annotations]]` table represents an association of Copyright and
+Licensing Information to zero or more Covered Files. It has the following keys:
+
+- `path` (REQUIRED), a string or list of strings representing paths. A path MUST
+  use forward slashes as path separators. A path SHOULD resolve to one or more
+  Covered Files relative to the `REUSE.toml` file's directory. A path MAY use
+  globbing to match several Covered Files in a single expression. These are the
+  globbing and matching rules:
+  - `*` matches everything except forward slashes (i.e. path separators).
+  - `**` and `**/` match everything including forward slashes (i.e. path
+    separators).
+  - In order to escape an asterisk and include it verbatim, prefix it with `\\`.
+    You cannot prefix it with `\` because that is invalid TOML. In order to
+    include a backslash verbatim, use `\\\\`. `\\` followed by any other
+    character is functionally equal to just typing that character.
+- `precedence` (OPTIONAL), a literal string. It determines the order of
+  precedence for Copyright and Licensing Information between the `REUSE.toml` file
+  and Covered Files in the table, and between multiple `REUSE.toml` files if they
+  both contain Information for the same Covered File. Available values are:
+  - `closest`, the default value when `precedence` is not defined. This is an
+    instruction to associate the Copyright and Licensing Information inside of
+    the Covered Files, if available. If no such Information is found, then the
```

Comment:
It may be worth to explicitely mention the .license files here. I know, above they are described as an extension to the inside of the covered files, but it may not be as obvious to the common reader.

Reply:
agree